tests/bootstrap.php

```diff
@@ -38,17 +38,15 @@
  */
 
 /* bootstrap composer's autoloader */
-require_once __DIR__ . '/../vendor/autoload.php';
+require_once dirname(__FILE__) . '/../vendor/autoload.php';
```

Comment:
We should prefer the magic constant ```__DIR__``` ([benchmark](https://beeznest.com/blog/2017/04/14/dirname-vs-dir-constant-which-is-faster/)) since we don't need to support PHP 5.3.

```suggestion
require_once __DIR__ . '/../vendor/autoload.php';
```